Cargo.toml

```
@@ -17,8 +17,8 @@ rev = "0c0cae6fa22b751e7fd95d7ce7556ac6d7b7010e"
 
 [workspace.dependencies.soroban-spec]
 version = "0.8.5"
-git = "https://github.com/stellar/rs-soroban-sdk"
-rev = "7d143e425b690f2981ae804ce249462d98a68b0d"
+git = "https://github.com/ahalabs/rs-soroban-sdk"
```

This should be stellar.
```suggestion
git = "https://github.com/stellar/rs-soroban-sdk"
```